chemicaltoolbox/openduck/openduck_system_prep.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="openduck_system_prep" name="OpenDUck system preparation" version="@VERSION@">
+    <description>for dynamic undocking</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        #set OPENDUCK_GPU_PARAM=1
```

Comment:
the above, you are setting here a cheetah variable. But you want to have an evironment variable `\${OPENDUCK_GPU_PARAM:-1}` --> note the backslash.